Formula/c/comtrya.rb

```diff
@@ -0,0 +1,14 @@
+class Comtrya < Formula
+  desc "Configuration and dotfile management tool"
+  homepage "https://comtrya/dev"
+  url "https://github.com/comtrya/comtrya/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "a5401004a92621057dab164db06ddf3ddb6a65f6cb2c7c4208a689decc404ad4"
+  license "MIT"
+  head "https://github.com/comtrya/comtrya.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "./app")
+  end
```

Comment:
you also need test block for the formula

Reply:
Added a test block, let me know if that is sufficient.